cranelift/codegen/src/isa/aarch64/abi.rs

```diff
@@ -264,13 +267,24 @@ impl ABIMachineSpec for AArch64MachineDeps {
                 *next_reg += 1;
                 remaining_reg_vals -= 1;
             } else {
-                // Compute size. Every arg takes a minimum slot of 8 bytes. (16-byte
-                // stack alignment happens separately after all args.)
+                // Compute the stack slot's size.
                 let size = (ty_bits(param.value_type) / 8) as u64;
-                let size = std::cmp::max(size, 8);
-                // Align.
+
+                let size = if call_conv != isa::CallConv::AppleAarch64 {
```

Comment:
When depending on new features of a dependency could you also update the dependency requirement in `Cargo.toml`? That would be useful for users like cg_clif that only update a single crate at a time and keep the rest pinned using `Cargo.lock`.